app.rb

```diff
@@ -770,4 +778,13 @@ class App < Sinatra::Base
     erb :about
   end
 
+  private
+
+  def pagy_get_vars(collection, vars)
```

Comment:
Needs minimal documentation that explains this is used for the pagination defaults.